### PR TITLE
WhenMainnet

### PR DESCRIPTION
**Pull request by Yostra**

### CODE REVIEW

File: chia/consensus/block_header_validation.py [hunk 1]
```diff
@@ -820,6 +820,7 @@ def validate_unfinished_header_block(
             prev_transaction_b = blocks.block_record(header_block.foliage_transaction_block.prev_transaction_block_hash)
             assert prev_transaction_b.timestamp is not None
             if header_block.foliage_transaction_block.timestamp <= prev_transaction_b.timestamp:
+                breakpoint()
```
Comment (mariano54) [inline chia/consensus/block_header_validation.py:823]: remove

File: chia/rpc/full_node_rpc_api.py [hunk 2]
```diff
@@ -63,8 +63,8 @@ async def _state_changed(self, change: str) -> List[WsRpcMessage]:
         return []
 
     async def get_initial_freeze_period(self):
-        freeze_period = self.service.constants.INITIAL_FREEZE_PERIOD
-        return {"INITIAL_FREEZE_PERIOD": freeze_period}
+        freeze_period = self.service.constants.INITIAL_FREEZE_END_TIMESTAMP
+        return {"INITIAL_FREEZE_END_TIMESTAMP": freeze_period}
```
Comment (mariano54) [inline chia/rpc/full_node_rpc_api.py:67]: @seeden needs to support this since warning is based on FREEZE_PERIOD

File: chia/rpc/wallet_rpc_api.py [hunk 3]
```diff
@@ -493,8 +493,8 @@ async def get_transactions(self, request: Dict) -> Dict:
         }
 
     async def get_initial_freeze_period(self):
-        freeze_period = self.service.constants.INITIAL_FREEZE_PERIOD
-        return {"INITIAL_FREEZE_PERIOD": freeze_period}
+        freeze_period = self.service.constants.INITIAL_FREEZE_END_TIMESTAMP
+        return {"INITIAL_FREEZE_END_TIMESTAMP": freeze_period}
```
Comment (mariano54) [inline chia/rpc/wallet_rpc_api.py:497]: @seeden

**Review by mariano54 — DISMISSED** [review]

File: chia/consensus/block_header_validation.py (hunk 1 above)
Comment (Yostra) [reply chia/consensus/block_header_validation.py:823]: yup, thanks for cathcing this

File: chia/rpc/full_node_rpc_api.py (hunk 2 above)
Comment (Yostra) [reply chia/rpc/full_node_rpc_api.py:67]: It does't look like it's used in main at the moment

**Pull request merged**